packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -192,6 +192,10 @@
   @apply flex items-center;
 }
 
+slot[name="actions-start"] {
+  block-size: 100%;
+}
```

Comment:
Is this working @josercarcamo? Do we need some new screenshot tests to account for these changes?

I would have expected either `align-items:stretch` or `align-self:stretch` needs to be used to stretch the items by either using the `::slotted` selector or some element within the shadow DOM of the component.

Something like: https://github.com/Esri/calcite-design-system/blob/b069284abb46dfd9b8384337f3825f5425f9d2b9/packages/calcite-components/src/components/list-item/list-item.scss#L78

or

https://github.com/Esri/calcite-design-system/blob/b069284abb46dfd9b8384337f3825f5425f9d2b9/packages/calcite-components/src/components/list-item/list-item.scss#L190-L193

Reply:
Yes, it is working, at least with manual testing. I tried the "align-items: stretch" but it didn't work in this case. I actually tried to use the solution in alert.css as shown below, but it didn't work.

`.actions-end {
  @apply flex self-stretch;
}`

Reply:
I'm just wondering why this is doing anything. A slot element is just a placeholder and really isn't an element that needs to be styled.